### ISSUE TITLE
seq2seq batch

### ISSUE THREAD

**Issue by ywk991112** [ywk991112/sentiment-chatbot#2]
### Pytorch Forums
[How can i compute seq2seq loss using mask?](https://discuss.pytorch.org/t/how-can-i-compute-seq2seq-loss-using-mask/861)
[How to use pack_padded_sequence in seq2seq models](https://discuss.pytorch.org/t/how-to-use-pack-padded-sequence-in-seq2seq-models/1293)
[RNN for sequence prediction](https://discuss.pytorch.org/t/rnn-for-sequence-prediction/182)
[How to use the backward functions for multiple losses?](https://discuss.pytorch.org/t/how-to-use-the-backward-functions-for-multiple-losses/1826)
[How to combine multiple criterions to a loss function?](https://discuss.pytorch.org/t/how-to-combine-multiple-criterions-to-a-loss-function/348)

### Github Examples
[MaximumEntropy/Seq2Seq-PyTorch](https://github.com/MaximumEntropy/Seq2Seq-PyTorch)
